plugins/system_upgrade.py

```diff
@@ -326,8 +327,8 @@ def __init__(self, base, cli):
 
 
 class SystemUpgradeCommand(dnf.cli.Command):
-    aliases = ('system-upgrade', 'fedup')
-    summary = _("Prepare system for upgrade to a new release")
+    aliases = ('system-upgrade', 'fedup', 'offline-upgrade', 'offline-distrosync',)
```

Comment:
This has a disadvantage of misleading usage message. And it's problematic because you can't use ``system-upgrade`` to do ``offline-upgrade`` due to the increased ``--releasever`` requirement. How about creating separate command class for each of these aliases that would inherit from this one? I am not sure if it's worth it, but I don't think it would be too much work. What do you think?

Reply:
Be honest that was my fist idea. But I rejected because it requires move nearly all code into pythonx-dnf-plugins-extra package to allow usage from multiple modules. But I got an extra idea how to do it without movement. But the only advantage will be in better information for command with --help.

Reply:
Resolved